dbms/src/Interpreters/TraceLog.cpp

```diff
@@ -0,0 +1,42 @@
+#include <Interpreters/TraceLog.h>
+#include <DataTypes/DataTypeArray.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <DataTypes/DataTypeDate.h>
+#include <DataTypes/DataTypeDateTime.h>
+
+namespace DB
+{
+
+Block TraceLogElement::createBlock()
+{
+    return
+    {
+        {std::make_shared<DataTypeDate>(),                                    "event_date"},
+        {std::make_shared<DataTypeDateTime>(),                                "event_time"},
+        {std::make_shared<DataTypeString>(),                                  "query_id"},
+        {std::make_shared<DataTypeArray>(std::make_shared<DataTypeUInt64>()), "trace"}
+    };
```

Comment:
TODO: add a function `symbolizeAddress` to print stack traces.
Probably we should use it to form a MATERIALIZED field with symbolized stack traces.